backend/models/Node.js

```diff
@@ -9,11 +9,11 @@ module.exports = (sequelize, DataTypes) => {
       },
       moniker: {
         type: DataTypes.STRING,
-        allowNull: false,
+        allowNull: true,
       },
       accountId: {
         type: DataTypes.STRING,
-        allowNull: false,
+        allowNull: true,
```

Comment:
It makes sense to just put an empty string if we don't get a string value on telemetry ping.

Reply:
I am not very sure how it works to fix #276 

@frol

Reply:
https://github.com/near/near-explorer/pull/283